.drone.yml

```diff
@@ -2474,409 +2474,389 @@ volumes:
   temp: {}
 
 ---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/mac.go:18
+################################################
+
 kind: pipeline
 type: exec
 name: build-darwin-amd64
-
-concurrency:
-  limit: 1
-
-platform:
-  os: darwin
-  arch: amd64
-
 trigger:
   event:
+    include:
     - tag
   ref:
     include:
-      - refs/tags/v*
+    - refs/tags/v*
   repo:
     include:
-      - gravitational/*
-
+    - gravitational/*
 workspace:
   path: /tmp/build-darwin-amd64
-
+platform:
+  os: darwin
+  arch: amd64
 clone:
   disable: true
-
+concurrency:
+  limit: 1
 steps:
-  - name: Set up exec runner storage
-    environment:
-      WORKSPACE_DIR: /tmp/build-darwin-amd64
-    commands:
-      - set -u
-      - mkdir -p $WORKSPACE_DIR
-      - chmod -R u+rw $WORKSPACE_DIR
-      - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
-
-  - name: Check out code
-    environment:
-      GITHUB_PRIVATE_KEY:
-        from_secret: GITHUB_PRIVATE_KEY
```

Comment:
As per above `GITHUB_PRIVATE_KEY` is needed here (good thing about `dronegen` is that changing it for one pipeline will fix it for all of them)